core/debug/inc/debug.h

```diff
@@ -33,6 +44,10 @@ void debug_fault_mpu(void);
 void debug_fault(THaltError reason, uint32_t lr, uint32_t sp);
 void debug_mpu_config(void);
 void debug_map_addr_to_periph(uint32_t address);
+void debug_register_box(const TUvisorDebugDriver * const driver);
```

Comment:
I would keep `debug_register_box` as it is used to register box number X as a debug box. The driver is a requirement in this process.